parser/goyacc/main.go

```diff
@@ -359,34 +359,6 @@ func main1(in string) (err error) {
 	mustFormat(f, "// CAUTION: Generated file - DO NOT EDIT.\n\n")
 	mustFormat(f, "%s", injectImport(p.Prologue))
 	mustFormat(f, `
-func init() {`)
-	for r, rule := range p.Rules {
-		if rule.Action == nil {
-			continue
-		}
-
-		action := rule.Action.Values
-		if len(action) == 0 {
-			continue
-		}
-
-		if len(action) == 1 {
-			part := action[0]
-			if part.Type == parser.ActionValueGo {
-				src := part.Src
-				src = src[1 : len(src)-1] // Remove lead '{' and trail '}'
-				if strings.TrimSpace(src) == "" {
-					continue
-				}
-			}
-		}
-		mustFormat(f, `
-                funcsSet[%d] = %s
-`, r, fmt.Sprintf("fnR%d", r))
-	}
-	mustFormat(f, `
-}
-var funcsSet []func(yyLexer, *Parser, int, []%[1]sSymType) = make([]func(yyLexer, *Parser, int, []%[1]sSymType), 2048)
```

Comment:
maybe better build 

`var funcSet = []func(yyLexer, *Parser, int, []%[1]sSymType){nil, nil, fnR2...........}`

directly, to avoid over 2048 question.

Reply:
Changes were undone because of potential side effect on performance.